dist/api/index.js

```diff
@@ -8,20 +8,26 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         step((generator = generator.apply(thisArg, _arguments || [])).next());
     });
 };
+var __importDefault = (this && this.__importDefault) || function (mod) {
+    return (mod && mod.__esModule) ? mod : { "default": mod };
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+// @ts-nocheck disable type-checking for now. turn me back on when feeling brave.
 /**
  * Account level API request definitions.
  */
-const fetch = require('isomorphic-fetch');
-const Token = require('./token');
-const utils = require('../utils/index');
-const { validateRequestAsJSON, checkStatus } = require('../utils');
-const { DEFAULT_API_URL } = require('../utils/constants');
-// REMOVEME
-const { thisIsATest } = require('./tsIntegrationTest');
+const isomorphic_fetch_1 = __importDefault(require("isomorphic-fetch"));
+const realmRoles_1 = require("./realmRoles");
+const token_1 = __importDefault(require("./token"));
```

Comment:
Out of curiosity why all of the `_1`

Reply:
ha a great question. compiled variable names are a box of mystery i have not yet had a reason to open. the important thing is that it will maintain the names of exports and that these seem to be entirely just internal implementation details.